compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/IOSSkikoInput.uikit.kt

```diff
@@ -21,11 +21,11 @@ import androidx.compose.ui.unit.DpOffset
 
 internal interface IOSSkikoInput {
 
-    fun beginFloatingCursor(offset: DpOffset) {}
+    fun beginFloatingCursor(offset: DpOffset)
 
-    fun updateFloatingCursor(offset: DpOffset) {}
+    fun updateFloatingCursor(offset: DpOffset)
 
-    fun endFloatingCursor() {}
+    fun endFloatingCursor()
```

Comment:
Removed default no-ops i introduced earlier to avoid future issues, like in this case with forgotten default method of `PlatformTextInputService`